frontend/src/app/firewood/forests/order-confirmation/order-confirmation.component.html

```diff
@@ -54,40 +51,40 @@ <h2 class="usa-heading">Order summary</h2>
       <h2 class="usa-heading">Order details</h2>
       <hr class="header-divider">
       <table class="tree-confirmation-table">
+        <div class="firewood-order-detail usa-width-one-third">
+          <dl>
+            <dt><strong>Quantity</strong></dt>
+            <dd>{{permit.quantity}}</dd>
+          </dl>
+        </div>
         <tr class="firewood-order-detail">
           <td><strong>Permit number:</strong></td>
-          <td>234</td>
-          <!-- <td>{{permit.permitNumber | uppercase}}</td> -->
+          <td>{{permit.permitNumber | uppercase}}</td>
         </tr>
         <tr class="firewood-order-detail">
           <td><strong>Name:</strong></td>
-          <td>Firstname Lastname</td>
-          <!-- <td>{{permit.firstName}} {{permit.lastName}}</td> -->
+          <td>{{permit.firstName}} {{permit.lastName}}</td>
         </tr>
         <tr class="firewood-order-detail">
           <td><strong>Payment:</strong></td>
-          <td>$20</td>
-          <!-- <td>${{permit.totalCost}}</td> -->
+          <td>${{permit.totalCost}}</td>
         </tr>
         <tr class="firewood-order-detail">
           <td><strong>Transaction method:</strong></td>
           <td>Credit card</td>
         </tr>
         <tr class="firewood-order-detail">
           <td><strong>Transaction date:</strong></td>
-          <td>1/2/30</td>
-          <!-- <td>{{permit.transactionDate | date: 'shortDate'}}</td> -->
```

Comment:
All of these changes are out of scope as well and will be addressed after I verify results on staging. Please omit.